automod2/automod2.py

```diff
@@ -71,7 +70,9 @@
 
 You can also prevent users from spamming images using [p]automod2 imagelimit
 """
-
+EMOJIS = {
+    'tips': ['\N{THUMBS UP SIGN}', '\N{THUMBS DOWN SIGN}', '\N{EYES}']

```

Comment:
These should be one per line, and include a trailing comma on the last element.

Reply:
fixed i thought python didnt allow trailing commas omg ive been never putting them and so sad about it